templates/base.html

```diff
@@ -37,15 +38,15 @@
         {% include "includes/footer.html" %}
     </footer>
 
-<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
-    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous">
-</script>
+<script src="https://code.jquery.com/jquery-1.12.4.js"></script>

```

Comment:
What is the purpose of going back from jQuery version 3.5.1 over to a much older version 1.12.4?

Reply:
we moved from jquery to bootstrap and resolve this issue.

Reply:
@alexeykuz-sys can we change it to this please `<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>`. That's a newer version, but not slim so it should do the trick.